src/shipit_workflow/shipit_workflow/models.py

```diff
@@ -134,6 +134,7 @@ def generate_phases(self, partner_urls=None, github_token=None):
             action_task_input['previous_graph_ids'] = list(previous_graph_ids)
             action_task_input['release_promotion_flavor'] = phase['name']
             action_task_id, action_task, context = generate_action_task(
+                decision_task_id=self.decicion_task_id,
```

Comment:
```
self.decicion_task_id
         ^
    *decision ?
```

Reply:
That is an existing typo in the parameter.